### DISCUSSION TITLE
Can it find the steady-state solutions of any Hamiltonian?

### DISCUSSION THREAD

**Discussion by balabalabalalaba** [qutip/qutip#1749] [Q&A] [answered]
If the Hamiltonian depended on the density matrix of state, can I find the steady state solution of it?

**Comment by balabalabalalaba** [comment]
If the system has a feedback, it would have a Hamiltonian which depended on the state.

**Comment by hodgestar** [comment, marked as answer]
I believe the short answer is currently "no", but evolving systems with feedback is something that we are including support for in QuTiP 5. If you can give a more precise description of the problem you'd like to solve, perhaps I or someone else can offer more suggestions.
